java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2351,6 +2352,20 @@ public T objectRefcount(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Updates the last access time of specified <code>keys</code> and returns the count of keys that
+     * were updated.
```

Comment:
Maybe shorten that? We have a return section to tell what command returns.
```suggestion
     * Updates the last access time of specified <code>keys</code>.
```

Reply:
Sure I can do that.